analyses/telomerase-activity-prediction/RUN-telomerase-activity-prediction.sh

```diff
@@ -0,0 +1,21 @@
+#enviroment settings
+set -e
+set -o pipefail
+
+# This script should always run as if it were being called from
+# the directory it lives in.
+script_directory="$(perl -e 'use File::Basename;
+  use Cwd "abs_path";
+  print dirname(abs_path(@ARGV[0]));' -- "$0")"
+cd "$script_directory" || exit
+
+#generate collapsed data for count files 
+Rscript ../analyses/collapse-rnaseq/01-summarize_matrices.R -i ../../data/pbta-gene-counts-rsem-expected_count.stranded.rds -g ../../data/gencode.v27.primary_assembly.annotation.gtf.gz -m ../analyses/collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed.stranded.rds -t ../analyses/collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed_table.stranded.rds
+Rscript ../analyses/collapse-rnaseq/01-summarize_matrices.R -i ../../data/pbta-gene-counts-rsem-expected_count.polya.rds -g ../../data/gencode.v27.primary_assembly.annotation.gtf.gz -m ../analyses/collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed.polya.rds -t ../analyses/collapse-rnaseq/pbta-gene-counts-rsem-expected_count-collapsed_table.polya.rds
+
+
+Rscript --vanilla 01-run-EXTEND.R --input ../../data/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds --output results/TelomeraseScores_PTBAStranded_FPKM.txt
```

Comment:
This failed for me locally because there is nothing in the Rscript that creates the `results` folder. You can add
```
mkdir -p results
```
to the shell script prior to running any of the `01-run-EXTEND.R` steps and it will create one for you.

Reply:
Thanks. We will do the changes. And we have a question regarding the comparison scripts. Could we hard code the path of files with file names for those and run those scripts from an analysis shell script (not containing input information)

Reply:
Can we provide the results for EXTEND scores from #489 as we don't have the annotation files to generate the collapsed files for counts data locally.

Reply:
If you hardcode file names in an Rscript, please use the `rprojroot` package to identify the root of the repository and code your paths relative to the root of the repository. This makes it such that if someone wanted to run a single step outside the context of the shell script, they could do that from anywhere in this project without issue. You can see an example of how to use this package in another part of the project here: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/eadb78d9ef02d35ae06a88104b29b4d0325499a6/analyses/transcriptomic-dimension-reduction/scripts/run-dimension-reduction.R#L53

Reply:
If you ran the download script with `bash download-data.sh`, you should have a local copy of the annotation file, see this part of the download script: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/eadb78d9ef02d35ae06a88104b29b4d0325499a6/download-data.sh#L49

Reply:
Sure. We will do as mentioned. Thanks